### DISCUSSION TITLE
pretrained model

### DISCUSSION THREAD

**Discussion by miaomiaomiao66** [huggingface/diffusers#10265] [Q&A]
I want to use latent diffusion for image generation task, a pretrained model autoencoderKL is required, how can I find the checkpoint file?

**Comment by suzukimain** [comment]
hi @miaomiaomiao66 , Is this helpful?

```python
from diffusers import AutoencoderKL

vae = AutoencoderKL.from_pretrained("CompVis/ldm-text2im-large-256", subfolder="vqvae")
```
